arcgis-ios-sdk-samples/Maps/Map loaded/MapLoadedViewController.swift

```diff
@@ -16,60 +16,56 @@ import UIKit
 import ArcGIS
 
 class MapLoadedViewController: UIViewController {
+    /// The map displayed in the map view.
+    let map = AGSMap(basemap: .imageryWithLabels())
 
-    @IBOutlet var mapView:AGSMapView!
-    @IBOutlet var bannerLabel:UILabel!
-    
-    private var map:AGSMap?
+    @IBOutlet var mapView: AGSMapView!
+    @IBOutlet var bannerLabel: UILabel!
     
     override func viewDidLoad() {
         super.viewDidLoad()
 
         //setup source code bar button item
-        (self.navigationItem.rightBarButtonItem as! SourceCodeBarButtonItem).filenames = ["MapLoadedViewController"]
-        
-        //initialize map with basemap
-        self.map = AGSMap(basemap: AGSBasemap.imageryWithLabels())
+        (navigationItem.rightBarButtonItem as! SourceCodeBarButtonItem).filenames = ["MapLoadedViewController"]
```

Comment:
would it be safer or better in any way to do this instead:

```swift
(navigationItem.rightBarButtonItem as? SourceCodeBarButtonItem)?.filenames = ["MapLoadedViewController"]
```

Reply:
Maybe, but all of the other samples use a force cast. 

I'd much rather refactor how we present the source code view. It is very odd that we are giving the button an array of file names.

Reply:
ok - I'm fine with entering an issue for that. However let's focus on getting our sample coverage up to par first.